### PR TITLE
[security] Fix `npm audit` vulnerabilities;

### PR DESCRIPTION
**Pull request by aleonov-virtru**
It's critical for our CI process to fix all `npm audit` issues so I would like to propose this dependencies update that will help us to continue using this library.

### CODE REVIEW

File: src/when.test.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 const { stringContaining } = expect
 
 const errMsg = ({ expect, actual }) =>
-  new RegExp(`Expected.*\\n.*${expect}.*\\nReceived.*\\n.*${actual}`)
+  new RegExp(`Expected.*${expect}.*\\nReceived.*${actual}`)
```
Comment (aleonov-virtru) [inline src/when.test.js]: `jest` changes error message output format.

File: src/when.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const assert = require('assert')
-const utils = require('expect/build/jasmine_utils')
+const utils = require('expect/build/jasmineUtils')
```
Comment (aleonov-virtru) [inline src/when.js]: `expect` changed jasmine utils placement in major version chagne

**Comment by timkindberg** [comment]
v2.6.0 - THANK YOU!

**Comment by aleonov-virtru** [comment]
Thanks much for quick reply!
Nave a nice day

**Pull request merged**